AppHub/AppHub/AHBuild.m

```diff
@@ -37,7 +37,7 @@ - (instancetype)initWithBundle:(NSBundle *)bundle info:(NSDictionary *)info
     if ((self = [super init])) {
         _buildDescription = [info[AHBuildDataDescriptionKey] copy] ?: @"This build was downloaded from the App Store.";
         _bundle = bundle ?: [NSBundle mainBundle];
-        _compatibleIOSVersions = info[AHBuildDataCompatibleIOSVersionsKey] ?: AHShortVersionString();
+        _compatibleIOSVersions = [info[AHBuildDataCompatibleIOSVersionsKey] allValues] ?: @[AHShortVersionString()];
```

Comment:
The issue right now is that it's exposed to api consumers as a `NSArray *` in the public header, and causes crash. We can change that to `NSDictionary *`, but presumably that will change the public api?